ingest_python_scripts/check_ingest.py

```
@@ -0,0 +1,647 @@
+import argparse
+import base64
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+from datetime import datetime
+
+# ---------- config ----------
+INSTANCE_ID = _env.get("INGEST_INSTANCE_ID", "")
```

Ruff (0.15.12)

[error] Undefined name `_env`

(F821)